code-challenges.test.js

```diff
@@ -26,37 +26,171 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // Pseudo code:
+// Input: Given objects with numbers 
+// Output : the number inside the arguement is evenly divisible by 3 or not
+// Create a test with expect statements for each of the variables provided.
+// // Process : Create a function called evenThree that will call an object as an arguement, add the string `is divisible by`, use modulo operand for division and set 3 strictly equal to a remainder of 0.
```

Comment:
Your code is shows that you created a class `Divide` to produce objects, but it is not reflected here in your pseudo code.